theme/static/js/custom.js

```diff
@@ -157,6 +187,65 @@ $(document).ready(function () {
         return false
     });
 
+    $("#hs-nav-bar .res-dropdown ul > li>  a").on("click", function () {
+        if ($("#select-resource-type").attr("data-anonymous")) {
```

Comment:
This is a bit hacky, routing to my-resource after logging in before creating the resource. We could just go straight to create-resource after login, but my guess is that's not setup for you right now.

Reply:
Correct. This logic could be added to the backend where we process this AJAX request. @pkdash, what do you think?

Reply:
This behavior was removed as part of recent changes.